SU-SWS/stanford_profile
# D8CORE-970: adding wysiwyg scss file

# READY FOR REVIEW 

# Summary
Changing the link styles in wysiwyg paragraph
_See ticket:  https://stanfordits.atlassian.net/browse/D8CORE-970_

# Needed Review By (Date)
Before RC1

# Urgency


# Steps to Test

- Checkout this branch
- Create a page with headings with links
- Verify you see:
![Screen Shot 2019-12-19 at 5 23 31 PM](https://user-images.githubusercontent.com/284440/71222196-642e3a00-2284-11ea-85d4-bc31559c062e.png)


# Affected Projects or Products
- D8Core (https://github.com/SU-SWS/acsf-cardinalsites)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/D8CORE-970

## Related PRs

## Accessibility Testing

## More Information

## Folks to notify
`@sherakama` 
`@kerri-augenstein`

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
